pkg/utils/miniojob/types.go

```diff
@@ -172,13 +172,17 @@ func (jobCommand *MinIOIntervalJobCommand) createJob(_ context.Context, _ client
 		},
 	}
 	baseEnvFrom = append(baseEnvFrom, jobCommand.CommandSpec.EnvFrom...)
+	urlProtocol := "http://"
+	if isTLS {
+		urlProtocol = "https://"
+	}
```

Comment:
Please use appropriate naming. The `http` / `https` part is called the _scheme_:
```
	scheme := "http"
	if isTLS {
		scheme = "https"
	}
```